sortbootorder.c

```diff
@@ -74,14 +74,15 @@ int main(void) {
 	u8 bootlist_map_ln = 0;
 	char *ipxe_str;
 	char *scon_str;
+	char *usb_str;
 
 #ifdef CONFIG_USB /* this needs to be done in order to use the USB keyboard */
 	usb_initialize();
 	noecho(); /* don't echo keystrokes */
 #endif
 
 	printf("\n****************************************************");
-	printf("\n*** Sortbootorder payload    ver 1.2   PC Engines  *");
+	printf("\n*** Sortbootorder payload    ver 1.3   PC Engines  *");
```

Comment:
@macpijan version bump should be as separate commit. We should have commit per logical change.